src/components/transaction-list/TransactionList.js

```diff
@@ -271,9 +270,6 @@ export default function TransactionList({
     e => {
       const { selection } = e.nativeEvent;
       switch (selection) {
-        case 'newimage':
-          onAvatarChooseImage();
```

Comment:
we are removing this completely now? has this been agreed on?

Reply:
i agree with jake, don't think we should be removing this one